bluebinder.c

```diff
@@ -368,38 +369,49 @@ bluebinder_callbacks_transact(
     if (!g_strcmp0(iface, BINDER_BLUETOOTH_SERVICE_IFACE_CALLBACKS)) {
         if (code == 1) { /* initializationComplete */
             int result = 0;
+            GBinderLocalReply* reply = NULL;
 
             gbinder_remote_request_read_int32(req, &result);
 
-            if (result != 0) {
-                fprintf(stderr, "Bluetooth binder service failed\n");
-                g_main_loop_quit(proxy->loop);
+            do{
+
```

Comment:
Missing space between `do` and `{` followed by an empty line looks like random typing to me :/ Some sort of consistency would be appreciated.

Reply:
sure, I'll fix